### PR TITLE
Replace deprecated <uri> for <url>

### PR DESCRIPTION
**Pull request by lahmatiy**
> ... CSS Level 3 came back to the initial definition, this time explicitly defined. The functional notation url() denotes a <url> CSS data type and no more the more generic <uri> CSS data type.

https://developer.mozilla.org/en-US/docs/Web/CSS/url

### CODE REVIEW

**Comment by teoli2003** [comment]
Some of these uri -> url has not made it to the spec [some will not directly be changed, because the syntax will be extended in the future (list-style-image will use <image> in the future)

But I think we should go ahead with these changes.

File: css/properties.json [hunk 1]
```diff
@@ -2792,7 +2792,7 @@
     "status": "experimental"
   },
   "content": {
-    "syntax": "normal | none | [ &lt;string&gt; | &lt;uri&gt; | &lt;counter&gt; | attr() | open-quote | close-quote | no-open-quote | no-close-quote ]+",
+    "syntax": "normal | none | [ &lt;string&gt; | &lt;url&gt; | &lt;counter&gt; | attr() | open-quote | close-quote | no-open-quote | no-close-quote ]+",
```
Comment (SebastianZ) [inline css/properties.json]: FYI, CSS 2.2 still [defines it using `<uri>`](https://www.w3.org/TR/CSS22/generate.html#propdef-content), but [it was resolved](https://lists.w3.org/Archives/Public/www-style/2016Oct/0091.html) to refer to [CSS Syntax](https://drafts.csswg.org/css-syntax/) instead, which removed `<uri>`. So it's fine that it gets changed already.

Sebastian

File: css/properties.json [hunk 2]
```diff
@@ -4074,7 +4074,7 @@
     "status": "standard"
   },
   "list-style-image": {
-    "syntax": "&lt;uri&gt; | none",
+    "syntax": "&lt;url&gt; | none",
```
Comment (SebastianZ) [inline css/properties.json]: Same as for `content`.

Sebastian

**Comment by SebastianZ** [comment]
> Some of these uri -> url has not made it to the spec [some will not directly be changed, because the syntax will be extended in the future (list-style-image will use in the future)

I've provided some more details in my review. The W3C already [resolved on changing CSS 2.2 property syntaxes to refer to CSS Syntax](https://github.com/w3c/csswg-drafts/issues/412). See https://github.com/w3c/csswg-drafts/issues/412.

Sebastian

**Pull request merged**